cpp/ql/src/Likely Bugs/Format/WrongTypeFormatArguments.ql

```diff
@@ -170,7 +170,8 @@ where
   ) and
   not arg.isAffectedByMacro() and
   not arg.isFromUninstantiatedTemplate(_) and
-  not actual.getUnspecifiedType() instanceof ErroneousType
+  not actual.getUnspecifiedType() instanceof ErroneousType and
+  not arg.(Call).getTarget().getADeclarationEntry().isImplicit()
```

Comment:
I am curious about whether this line of logic should be encapsulated a bit better. In C++ we would get an `ErroneousType` but in C, implicit function declarations return an `int`.